pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = neighborhoods_backend.settings
```

Comment:
This needs to match the path to the settings in the exemplar (i.e. `dead_songs`)

Reply:
right! I left this as it was because this is one file that wasn't present in the backend exemplar so I wanted to make that clear by not making edits until the decision is made if the file should be included.

Reply:
pretty sure `pytest.ini` is needed when we switch to the `pytest` framework for testing - if not absolutely necessary, I have a feeling if it came from Brian or Nathan then it's a Good Idea.

Reply:
@MikeTheCanuck Yes, its a required configuration file for using pytest, which is what @BrianHGrant  and finally converged on as the best unit testing framework for our backends. I would advise that we port that to the backend-exemplar along with my working unit test cases.

Reply:
roger. My sense is that this PR is very unlikely to be merged.

Reply:
certainly not in it's current form.